azerothcore/mod-npc-gambler
# Update SQL and change NPC cosmetics.

Changes Proposed:
Update SQL
Issues Addressed:
Fixes failure and issues related to changes to Creature_Template
Cosmetic Change:
Changed NPC to Elf Female "Dobbo Girl" Named Loita 
(Blizzardlike play on words nod to the Character Leeta on Star Trek Deep Space Nice)

Tests Performed:
Tested on Current Standard Build and Also Playerbots
How to Test the Changes:
Install Module
Should be no errors upon compile and install
SQL should import without errors
Summon NPC (Creature ID 601020)
Right click to speak to NPC - She Should allow you to gamble
